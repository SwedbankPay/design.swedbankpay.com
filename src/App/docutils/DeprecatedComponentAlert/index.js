import React from "react";
import Alert from "@components/Alert";

const DeprecatedComponentAlert = () => (
    <Alert type="danger">
        <h3 className="text-danger">Component deprecated!</h3>
        <p>This component has been deprecated and should not be used!</p>
    </Alert>
);

export default DeprecatedComponentAlert;
