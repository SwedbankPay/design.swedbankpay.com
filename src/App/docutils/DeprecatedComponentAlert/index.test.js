import React from "react";
import { shallow } from "enzyme";

import DeprecatedComponentAlert from "./index";

describe("Utilities: DeprecatedComponentAlert", () => {
    it("is defined", () => {
        expect(DeprecatedComponentAlert).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<DeprecatedComponentAlert />);

        expect(wrapper).toMatchSnapshot();
    });
});
