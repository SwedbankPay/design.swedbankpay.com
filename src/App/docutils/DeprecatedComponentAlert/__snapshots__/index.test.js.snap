// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utilities: DeprecatedComponentAlert renders 1`] = `
<Alert
  type="danger"
>
  <h3
    className="text-danger"
  >
    Component deprecated!
  </h3>
  <p>
    This component has been deprecated and should not be used!
  </p>
</Alert>
`;
